doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -455,6 +455,9 @@ Bluetooth Fast Pair samples
     * An application versioning using the :file:`VERSION` file.
     * The DFU support which can be enabled using the ``SB_CONFIG_APP_DFU`` sysbuild Kconfig option.
       DFU is available for all supported targets except the ``debug`` configurations of :ref:`zephyr:nrf52dk_nrf52832` and :ref:`zephyr:nrf52833dk_nrf52833` due to size constraints.
+    * An application module for the DULT motion detector feature and the new UI for supported board targets to interact with this feature.
```

Comment:
Remove the changelog from this PR and consult @divipillai . She will add the entries to her PR.

Reply:
Yes, I just wanted to get approves here and later move it to Divya's PR.

Reply:
Please remove DNM label when it's done, it could've been merged by accident.

Reply:
Done